### PR TITLE
tools/cephfs-shell: pep8 clean

### PR DESCRIPTION
**Pull request by tchaikov**
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

**Comment by tchaikov** [comment]
just for early review, will test it and then remove the DNM label.

File: src/tools/cephfs/cephfs-shell [hunk 1]
```diff
@@ -205,30 +234,28 @@ def copy_to_local(shell, remote_path, local_path):
...
+            shell.poutput(file_chunk.decode('utf-8'))
     cephfs.close(file_)
-    if fd: 
-        fd.close()   
+    if fd:
+        fd.close()
+
 
-def dirwalk(dir_name, giveDirs=0):
+def dirwalk(dir_name):
```
Comment (rpavani1998) [inline src/tools/cephfs/cephfs-shell]: This method now, only walks into directories which has no sub-directories.

File: src/tools/cephfs/cephfs-shell [hunk 2]
```diff
@@ -215,10 +215,9 @@ def copy_from_local(shell, local_path, remote_path):
 
 
 def copy_to_local(shell, remote_path, local_path):
-    local_dir = os.path.dirname(local_path)
+    local_dir = os.path.dirname(local_path))
```
Comment (batrick) [inline src/tools/cephfs/cephfs-shell]: This looks like it's a syntax error?

**Comment by batrick** [comment]
Code seems to work. I'm going to merge this. Any fixes can be done separately.

**Pull request merged**